.goreleaser.yml

```diff
@@ -13,18 +13,35 @@ builds:
     dir: .
     main: ./main.go
     binary: postee
+    ldflags:
+      - -s -w
+      - "-extldflags '-static'"
+      - -X main.version={{.Version}}
     goos:
       - darwin
       - linux
     goarch:
       - amd64
-
+      - arm
+      - arm64
+    goarm:
+      - 7
 archives:
-  - name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}"
+  - name_template: "{{ .ProjectName }}_{{.Version}}_{{ .Os }}_{{ .Arch }}"
     builds:
       - postee
     replacements:
-      amd64: x86_64
+      amd64: 64bit
+      386: 32bit
+      arm: ARM
+      arm64: ARM64
+      ppc64le: PPC64LE
+      darwin: macOS
+      linux: Linux
+      openbsd: OpenBSD
+      netbsd: NetBSD
+      freebsd: FreeBSD
+      dragonfly: DragonFlyBSD
```

Comment:
This is cool but are we sure Postee can run on these platforms? I'm a little hesitant adding these without any obvious need or want for it.